sentry_sdk/integrations/celery.py

```diff
@@ -140,13 +140,21 @@ def apply_async(*args, **kwargs):
         # type: (*Any, **Any) -> Any
         hub = Hub.current
         integration = hub.get_integration(CeleryIntegration)
-        if integration is not None and integration.propagate_traces:
+
```

Comment:
We could just early return here if integration is None and save ourselves any future `integration is not None` checks below:
```suggestion
if integration is None:
    return f(*args, **kwargs)
```

Reply:
YES!